packages/okta-angular/README.md

```diff
@@ -32,9 +32,10 @@ All of these features are supported by this SDK. Additionally, using this SDK, y
 This library currently supports:
 
 - [OAuth 2.0 Implicit Flow](https://tools.ietf.org/html/rfc6749#section-1.3.2)
-- [OAuth 2.0 Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1) with [Proof Key for Code Exchange (PKCE)](https://tools.ietf.org/html/rfc7636) 
+- [OAuth 2.0 Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1) with [Proof Key for Code Exchange (PKCE)](https://tools.ietf.org/html/rfc7636)
 
 > This library has been tested for compatibility with the following Angular versions: 4, 5, 6, 7, 8, 9
+> :warning: Angular versions older than 7 may not be fully compatible with all dependencies of this library, due to an older Typescript version. You may be able to workaround this issue by setting `skipLibChecks: true` in your `tsconfig.json` file.
```

Comment:
Do we want to claim compatibility with all of these older versions?

Reply:
This is related to use of the "unknown" type within the typings for auth-js, "unknown" was introduced in Typescript 3.0. Angular 6 uses a 2.x version of Typescript so it complains. I don't think this breaks compatibility, but we might get questions about it.

Reply:
I wasn't talking about the details of that particular error, I'm just saying do we want to commit to supporting this set of versions.

Reply:
I say let's wait until a user has an issue and see if we can update the docs with a simple workaround. If not, we can drop support. I don't think there are that many angular users who fail to upgrade - they should expect problems if using a very old version mixed with current versions of other modules.